python/mxnet/optimizer.py

```diff
@@ -173,6 +182,36 @@ def create_state(self, index, weight):
             The state associated with the weight.
         """
 
+    def create_mp_state(self, index, weight):
+        """Creates auxiliary state for a given weight, including FP32 master
```

Comment:
including FP32 master copy if necessary.
->
including fp32 high precision copy if original weight is fp16